### PR TITLE
Features/pulling yelp data

### PR DESCRIPTION
**Pull request by stevensse21**
- Scrape restaurant reviews to generate fake users for ML training
- scrape the yelp website for data from attributes and store locally (sql)
- pull relevant Yelp API data (not based on user inputs, just retrieve data)/Scrape yelp website for user profile data to build out ML training data
(there are 2 cards with the same task)

### CODE REVIEW

File: yelp/UserYelpWebScraping.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                    occasion = 'date'
+                    num_people = 2
+                            
+                elif q=='solo':
+                    occasion = 'solo'
+                    num_people = '1'
+                            
+                elif q=='friend':
+                    occasion = 'friends'
+                    num_people = 2
```
Comment (menglander2019) [inline yelp/UserYelpWebScraping.py:131]: jc why is the default number of people for the 'friends' occasion 2?

**Review by menglander2019 — APPROVED** [review]
LGTM

File: yelp/UserYelpWebScraping.py (hunk 1 above)
Comment (stevensse21) [reply yelp/UserYelpWebScraping.py:131]: If the person in the review said friend (singular), we would still call the occasion "friends", but we know it was just the reviewer and the single friend. If the review said friends, I assume it was the reviewer, and 2 or more other friends and randomly generate a number (you can see in the next elif)

**Comment by stevensse21** [comment]
I am aware that I pushed the code with the API key visible. I will generate a new one and put it in the .gitignore at the beginning of the next sprint.

**Review by zblooms282 — APPROVED** [review]
LGTM

**Pull request merged**